packages/edit-site/src/index.js

```diff
@@ -77,11 +70,6 @@ export function initializeEditor( id, settings ) {
 		showListViewByDefault: false,
 	} );
 
-	dispatch( interfaceStore ).setDefaultComplementaryArea(
-		'core',
-		'edit-site/template'
-	);
```

Comment:
What I actually don't understand is why the sidebar was closed by default before. Because this code was in place even before my refactoring :)